src/components/modal/ModalStation/SelectStation/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { SpaceProps, space } from 'styled-system';
+import { Formik, Form } from 'formik';
+import Title from '../../../common/Title';
+import { StyledProps } from '../../../../types/theme';
+import Button from '../../../common/Button';
+import Paragraph from '../../../common/Paragraph';
+import ModalWindowButtonsContainer from '../../ModalWindow/ModalWindowButtonsContainer';
+import ContentContainer from '../../ModalWindow/ModalContentContainer';
+import FieldButton from '../../../common/FieldButton';
+import { IStationData } from '../../../../types/models/models';
+
+interface ISelectStationProps {
+  data: IStationData;
+  next: (values: Partial<IStationData>) => void;
+}
+
+const StyledButtonsContainer = styled.div<StyledProps & SpaceProps>`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ theme }) => theme.space[3]};
+  ${space}
+`;
+
+const StyledForm = styled(Form)`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`;
+
+function SelectStationContainer(props: ISelectStationProps) {
+  const { data, next } = props;
```

Comment:
It's a better practise to follow this pattern

```suggestion
const SelectStationContainer: React.FC<ISelectStationProps> => ({ data, next }) {
```